service/model_downloader.py

```diff
@@ -1,4 +1,5 @@
 from huggingface_hub import HfFileSystem, hf_hub_url, model_info
+from huggingface_hub.utils._errors import RepositoryNotFoundError
```

Comment:
this import is not resolved from my machine. On the installed service, the errors package seems to exist as "errors" (without underscore). I can simply import the Error from hugginface_hub.utils. 
Which verison of hugginface_hub are you using?

Reply:
This is true. It seems like there is a general version mismatch between the private and public fork. We were working with an other version...